src/pkg/error/errors.go

```diff
@@ -0,0 +1,184 @@
+package error
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+)
+
+var (
+	// ErrNotFound ...
+	ErrNotFound = errors.New("not found")
```

Comment:
Why we need these error definitions? I think the error code is enough to identify the error